static/templates/settings/deactivated_users_admin.hbs

```diff
@@ -1,11 +1,11 @@
 <div id="admin-deactivated-users-list" class="settings-section" data-name="deactivated-users-admin">
-    <div class="alert-notification" id="deactivated-user-field-status"></div>
     <div class="clear-float"></div>
```

Comment:
Is `clear-float` here supposed to move along with it?  I don't remember what that would be for but it seems like it could be related.